docs/README.md

```diff
@@ -66,11 +66,20 @@ You can submit chart related changes in three methods:
 1. Submit a chart without Chart Verification Report
 
    - Chart as a tarball
+      - must be created using ```helm package```
```

Comment:
I think this should be "**can** be created using `helm package`". A user can also do `tar -cvzf <target>.tgz <chart-directory>`.

Reply:
I think 'helm package' command should be recommended as by default the resulting tar file is of format <chart-name>-<chart-version>.tgz .While for tar command user have to specify the target tgz file name, which is prone to user error.

Reply:
@zonggen @tisutisu have update based on your suggestions.